instat/dlgOneWayFrequencies.vb

```diff
@@ -170,8 +152,8 @@ Public Class dlgOneWayFrequencies
     End Sub
 
     Private Sub ucrBase_BeforeClickOk(sender As Object, e As EventArgs) Handles ucrBase.BeforeClickOk
-        If rdoTable.Checked Then
-            ucrBase.clsRsyntax.SetBaseROperator(clsTableBaseOperator)
+        If rdoTable.Checked OrElse rdoBoth.Checked Then

```

Comment:
Yes I noticed that too for the graph. I think by resetting the assign to it might resolve it. I can check.